src/gateway/bridge/shard_runner.rs

```diff
@@ -42,8 +42,9 @@ pub struct ShardRunner {
     #[cfg(feature = "cache")]
     pub cache: Arc<Cache>,
     pub http: Arc<Http>,
+    // do not use mutex! this is used for almost every gateway event!
```

Comment:
Remove this comment, if there is a reason someone PRs this back to Mutex it will have to justify the performance hit anyway.

Reply:
Ok, I will remove it.